machine_common_sense/controller.py

```diff
@@ -56,6 +54,19 @@ def __image_depth_override(self, image_depth_data, **kwargs):
 ai2thor.server.Event._image_depth = __image_depth_override
 
 
+class NumpyAwareEncoderOverride(json.JSONEncoder):
+    # From https://github.com/allenai/ai2thor/blob/master/ai2thor/server.py#L17-L24 # noqa: E501
```

Comment:
Suggest changing this to a permalink so it doesn't go stale if the upstream repo changes
```suggestion
    # From https://github.com/allenai/ai2thor/blob/bd35d2cb887faee8b87aa04bd9373b027eb39f17/ai2thor/server.py#L17-L24 # noqa: E501
```